docs/MIGRATION-TO-ASYNC-AWAIT.md

```diff
@@ -864,7 +864,7 @@ e.g.
 const IpfsHttpClient = require('ipfs-http-client')
 const ipfs = IpfsHttpClient()
 
-const files = await ipfs.addFromURL('https://ipfs.io/images/ipfs-logo.svg')
+const files = await ipfs.addFromURL('https://example.com/images/ipfs-logo.svg')
```

Comment:
I assume here and other places we are ok to use a URL for a resource that doesn't exist.  If so then the PR seems good to me?  Everything else checked out.

Reply:
IANA [reserves this domain](https://www.iana.org/domains/reserved) specifically for use in documentation, but it might be nicer if the user could copy/paste this sort of example and have it work.  What do you think @lidel?

Reply:
If we ever reorganize `ipfs.io` and change path of logo, the example will no longer work (happened the last time ipfs.io was revamped).

It is better to use `example.com` than "real" websites: 
communicates well it is not a "live" example.